env/WCOSS2.env

```diff
@@ -227,5 +248,8 @@ elif [ $step = "gempak" ]; then
     [[ $NTHREADS_GEMPAK -gt $nth_max ]] && export NTHREADS_GEMPAK=$nth_max
     export APRUN_GEMPAKCFP="$launcher -np \$ntasks $mpmd"
 
+elif [ $step = "wafsgcip" ]; then
+
+    export MPIRUN="$launcher -np $npe_wafsgcip $mpmd"
```

Comment:
This is a cfp launcher setting (specifically $MPIRUN for the wafs script) so it should use `-np`.

Sample from the WCOSS2 documentation:
```
mpiexec -np 2 --cpu-bind verbose,core cfp ./command_file
```

In WCOSS2.env it sets `$mpmd` as:
```
export mpmd="--cpu-bind verbose,core cfp"
```
...so that `MPIRUN` becomes this at runtime:
```
MPIRUN='mpiexec -l -np 2 --cpu-bind verbose,core cfp'
mpiexec -l -np 2 --cpu-bind verbose,core cfp gcip.cmdfile
```